_posts/2022-12-01-trino-summit-2022-sk-telecom-recap.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Journey to Iceberg with Trino"
+author: "JaeChang Song, Jennifer Oh, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/apple.jpg
+---
+
+This post comes from [the later half of Trino Summit 2022 session]({% post_url 
+2022-11-21-trino-summit-2022-recap %}). Our friends JaeChang and Jennifer from
+SK Telecom, traveled across the globe from South Korea to join us in person! SK
+Telecom recently had some issues scaling Trino on the Hive model, among other
+issues that come with Hive. While some initial tweaking helped speed things up,
+it ultimately never solved the problem. SK Telecom initially tested performance
+after moving on Iceberg with some very impressive results. They talk about their
+journey to Iceberg via Trino.
+
+<!--more-->
+
+{% youtube V9_aPLXATh8 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@SK-Telecom.pdf">
+  Check out the slides here!
+</a>
+
+## Recap
+
+SK Telecom is a South Korean Telecom company that has built and operated
+open source based on-premise data platform for manufacturing data to determine
```

Comment:
```suggestion
an open source-based on-premise data platform for manufacturing data to determine
```

Reply:
It also might make sense to delete "manufacturing data" because we mention the data is for manufacturing yield right after, and it's easy to misread this as if they are manufacturing the data, not using it for data related to manufacturing.